doc/source/glossary.rst

```diff
@@ -77,4 +77,6 @@
 
   S3 compatible storage   
      This is the storage that is built on the :term:`S3 <Amazon S3>` API.
-   
\ No newline at end of file
+ 
+  Server-side encryption
+     Server-side encryption is the encryption of data on the remote storage. The data is encrypted when it is written to S3 bucket and decrypted when you access the data. 
```

Comment:
I think this first sentence might be interpreted as saying PBM can do encryption as it saves to the remote storage (including filesystem type).

I'm not sure what to say - does the following seem better?
```suggestion
     Server-side encryption is the encryption of data by the remote storage server as it receives it. The data is encrypted when it is written to S3 bucket and decrypted when you access the data. 
```

Reply:
Yes, Akira. Thank you for your suggestion